core-java/src/main/java/org/spine3/server/aggregate/AggregateRepositoryBase.java

```diff
@@ -131,7 +137,7 @@ private Method dispatchAsMethod() {
      */
     @Nonnull
     @Override
-    public A load(I id) throws IllegalStateException {
+    public A load(@Nonnull I id) throws IllegalStateException {
```

Comment:
Why do we need this annotation here. The ID cannot be `null` ever, and it's covered by default annotation at the package level.